articles/policies/rate-limits.md

```diff
@@ -319,3 +319,17 @@ For database connections, Auth0 limits certain types of repeat login attempts de
 ## Limits on SMS Messages for MFA
 
 There's a limit of 10 SMS messages/hour per user for multi-factor authentication. For more information, see [Configuring Twilio for SMS](/multifactor-authentication/twilio-configuration).
+
+## Limits on Sign In with Apple 
+
+The limits are the following:
+
+| Endpoint | Path | Limited by | Affected Tenants | Rate Limits |
+| --- | --- | --- | --- | --- |
+| Get Token	| `/oauth/token` | Native Social Login (Apple, Facebook, etc.) | Free |	30 requests per minute |
```

Comment:
The path is being split in multiple lines which is not consistent with other tables in the same page